docs/examples/sheetmobile/textOnlyHeader.js

```diff
@@ -0,0 +1,81 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import {
+  Layer,
+  SheetMobile,
+  Box,
+  Text,
+  Flex,
+  Icon,
+  DeviceTypeProvider,
+  Button,
+  FixedZIndex,
+  CompositeZIndex,
+} from 'gestalt';
+
+export default function Example(): Node {
+  const [showComponent, setShowComponent] = useState(true);
+  const PAGE_HEADER_ZINDEX: FixedZIndex = new FixedZIndex(10);
+  const ABOVE_PAGE_HEADER_ZINDEX: CompositeZIndex = new CompositeZIndex([PAGE_HEADER_ZINDEX]);
+
+  return (
+    <DeviceTypeProvider deviceType="mobile">
+      {showComponent ? (
+        <Layer zIndex={ABOVE_PAGE_HEADER_ZINDEX}>
+          <SheetMobile
+            align="center"
+            heading="Start creating now"
+            subHeading="Inspiration starts here"
+            onDismiss={() => setShowComponent(false)}
+            showDismissButton={false}
+            size="auto"
+          >
+            <Flex gap={12} height="100%" alignItems="center" justifyContent="center">
```

Comment:
How about gap={4} for this example. Looks a little off when the content overflows the header

Reply:
What do you mean>Screenshot?